backend/entities/Models/AgencyFilter.cs

```diff
@@ -0,0 +1,120 @@
+using Pims.Core.Extensions;
+using System;
+using System.Collections.Generic;
+
+namespace Pims.Dal.Entities.Models
+{
+    /// <summary>
+    /// AgencyFilter class, provides a model for filtering agencies.
```

Comment:
Not all these fields are used, should I just remove them? Thought they would be good to  have in case we want further filtering in the future.

Reply:
I don't think we should filter by anything we won't put an index on `Description`.  You also won't ever filter on `Id`.